gw_challenge/ingested_program/whitelist.txt

```diff
@@ -7,3 +7,14 @@ torch
 open-clip-torch
 torchvision
 transformers
+pandas
+Pillow
+scipy
+tqdm
+pyarrow
+matplotlib
+xgboost
+nflows
+lightgbm
+seaborn
+iminuit
```

Comment:
also include the following
numpy
scikit-learn
torchaudio
Keras
catboost
plotly

Reply:
For future reference, three dots at the top right (next to comment and view) have the option to "view file" and clicking on the blue bar in the file preview in this window will expand to show more since GitHub truncates to focus on the part that has been changed.